### PR TITLE
LPS-168865,LPS-168318

### PR DESCRIPTION
**Pull request by p-albuquerque**
Related issue: https://issues.liferay.com/browse/LPS-168865

**Summary:**
- Use objectDefinitionERC1 instead objectDefinitionId1;
- Use objectDefinitionERC2 instead objectDefinitionId2;
- Use parameterObjectFieldName instead parameterObjectFieldId;

All of these changes apply in ObjectRelationship Rest Layer/DTO and front end

### CODE REVIEW

**Comment by p-albuquerque** [comment]
ci:test:relevant

**Comment by p-albuquerque** [comment]
ci:test:sf

**Comment by p-albuquerque** [comment]
ci:test:relevant

**Comment by p-albuquerque** [comment]
ci:test:relevant

File: modules/apps/object/object-admin-rest-impl/src/main/java/com/liferay/object/admin/rest/internal/resource/v1_0/ObjectRelationshipResourceImpl.java [hunk 1]
```diff
@@ -169,11 +171,20 @@ public ObjectRelationship postObjectDefinitionObjectRelationship(
...
 		}
 
+		long parameterObjectFieldId = 0L;
+
+		ObjectField parameterObjectField =
+			_objectFieldLocalService.fetchObjectField(
+				objectDefinitionId2,
+				objectRelationship.getParameterObjectFieldName());
+
+		if (!Objects.isNull(parameterObjectField)) {
```
Comment (gabrielwas) [inline modules/apps/object/object-admin-rest-impl/src/main/java/com/liferay/object/admin/rest/internal/resource/v1_0/ObjectRelationshipResourceImpl.java:181]: @p-albuquerque change to `Validator.isNotNull` please

**Review by gabrielwas — CHANGES_REQUESTED** [review]

**Review by danvb112 — CHANGES_REQUESTED** [review]
Hey @p-albuquerque, Great job on this PR, I have just one suggestion about :smile:

**Comment by p-albuquerque** [comment]
Resent in: https://github.com/liferay-objects/liferay-portal/pull/1418

**Pull request closed without merging**